src/groups/groups.controller.ts

```diff
@@ -0,0 +1,38 @@
+import { Body, Controller, Delete, Get, Param, Post, Put } from "@nestjs/common";
+import { GroupsService } from "./groups.service";
+import { randomFill, randomUUID } from "crypto";
+import { createGroupDTO } from "./groups.dto";
+
+
+@Controller('group')
+export class GroupsController{
+    constructor(
+        private readonly groupService : GroupsService
+    ){}
+
+    @Post('/')
+    async createGroup(@Body('group') data : createGroupDTO,){
+        const uuid = randomUUID() ;
```

Comment:
can we mote this inside the createGroup function itself? we can take this as optional parameter and perform validation in function